app/admin/RTMediaUploadTermsAdmin.php

```diff
@@ -193,6 +193,20 @@ public function admin_setting_add_terms_option( $render_options, $options ) {
 		return $render_options;
 	}
 }
+/**
+ * Check if old plugin rtMedia upload terms is activate or not.
+ */
+
+function rtm_check_admin_upload_terms() {
+
```

Comment:
Remove the blank line above. The documentation must be exactly above the function declaration. @malavasita